src/config/media/pipewire.md

```diff
@@ -0,0 +1,87 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+PipeWire should be started as a user. Use the autostarting mechanism of your
+desktop environment.
+
+If using [startx](../graphical-session/xorg.md#startx), add the following line
+in your `.xinitrc` file:
+
+```
+pipewire &
+```
```

Comment:
This should be a note in the startx section about how to start arbitrary programs together with your X session.

Reply:
Thanks, I've added a note on this in the startx section. Let me know if it looks good to you.